wlog/svclog/svc1log/params.go

```diff
@@ -138,10 +140,15 @@ func OriginFromCallLineWithSkip(skipFrames int) Param {
 func initLineCaller(skip int) (string, int, bool) {
 	// the 1 skips the current "initLineCaller" function
 	_, file, line, ok := runtime.Caller(1 + skip)
-	if ok {
-		file = gopath.TrimPrefix(file)
+	if !ok {
+		return file, line, ok
 	}
-	return file, line, ok
+	cfg := &packages.Config{Mode: packages.NeedModule | packages.NeedName}
+	pkgs, err := packages.Load(cfg, "file="+file)
```

Comment:
See the docs for packages to see where this `file=...` pattern comes from
https://pkg.go.dev/golang.org/x/tools/go/packages